src/app/admin/duplicates/page.tsx

```
@@ -1,460 +1,299 @@
+								<div className="grid gap-3">
+									{detection.questions.map((q: any) => (
+										<div
+											key={q._id}
+											className={`group relative p-4 rounded-xl border-2 transition-all cursor-pointer ${keepQuestionId === q._id
+													? 'border-green-500 bg-green-500/5 ring-1 ring-green-500'
+													: selectedToDelete.has(q._id)
+														? 'border-red-200 bg-red-50/50 opacity-60'
+														: 'border-muted hover:border-primary/30'
+												}`}
+											onClick={() => handleSelectKeep(q._id, detection.questions.map((qu: any) => qu._id))}
+										>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)